package.json

```
@@ -92,10 +92,10 @@
     "react-scrollchor": "6.0.0",
     "react-select": "3.0.8",
     "react-stickynode": "2.1.1",
-    "react-stripe-elements": "6.1.0",
+    "react-stripe-elements": "6.0.1",
+    "react-swipeable": "^5.5.1",
```

Should stick with a fixed version

```suggestion
    "react-swipeable": "5.5.1",
```

---

@Betree  Done.